qiskit_experiments/library/randomized_benchmarking/clifford_utils.py

```diff
@@ -124,7 +124,7 @@ def _synthesize_clifford_circuit(
 ) -> QuantumCircuit:
     # synthesizes clifford circuits using given basis gates, for use during
     # custom transpilation during RB circuit generation.
-    return transpile(circuit, basis_gates=list(basis_gates), optimization_level=0)
+    return transpile(circuit, basis_gates=list(basis_gates), optimization_level=1)
```

Comment:
I think this `optimization_level` should be the taken from the `options` and not hard-coded as 1. The default level will be 1 because you set it in `_default_transpile_options`.

Reply:
We are going to remove `transpile_options` from RB experiments (and equip a new scheme to allow users how to synthesize Cliffords). So I'd like to decrease the dependence with `transpile_options` in the new path. That's why I decided to support only the default `optimization_level==1` here and use the old path for the case `optimization_level != 1`.